qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -148,6 +148,37 @@ def calibrations(self) -> Calibrations:
         """Return the calibrations."""
         return self._cals
 
+    @property
+    def analysis(self) -> Union[BaseAnalysis, None]:
```

Comment:
What's the reason to redefine here, if already defined in `BaseExperiment`?

Reply:
We can also do `@BaseExperiment.analysis.setter` but we cannot override only setter in the subclass as far as I understand.